src/context/AuthContext.tsx

```diff
@@ -23,7 +23,7 @@ type AuthContextProviderProps = {
 };
 
 export const AuthProvider = ({ children }: AuthContextProviderProps) => {
-  var [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
This has to be a var because a user can signin or signout and the state would change accordingly.

Reply:
udpated